shell/executor.go

```diff
@@ -227,8 +242,9 @@ var commandMap = map[string]Command{
 }
 
 var netCommands = map[string]Command{
```

Comment:
The same question of when to use := vs var declaration. Both are valid

Reply:
I do know that ":=" is not valid in this specific situation, i.e., it can't be used to declare globals.

Reply:
Did not know that. Noted and learned. Ten Golang tutorials didn't make me an expert?

Reply:
@johnmichaelwelsh That happens with the 11th.